src/main/java/pl/wojciechkostecki/revel/controller/LocalController.java

```diff
@@ -21,39 +22,39 @@ public LocalController(LocalService localService) {
     }
 
     @PostMapping
-    public ResponseEntity<Local> createLocal(@RequestBody Local local) {
-        logger.debug("REST request to create Local: {}",local);
-        Local savedLocal = localService.save(local);
+    public ResponseEntity<Local> createLocal(@RequestBody LocalDTO localDTO) {
```

Comment:
Why did you use dto here if it is exact copy of the entity? Whats the point of this DTO then?

Reply:
I have already corrected all DTOs